packages/plugin-javascript/__tests__/__snapshots__/index.js.snap

```diff
@@ -77,36 +77,36 @@ Array [
 
 exports[`javascript-universal resolves './modules/es6.symbol' without stripping .symbol suffix 1`] = `
 Array [
-  "{BASE_URL}modules/es6.symbol.js",
-  "{BASE_URL}modules/es6.symbol/index.js",
-  "{BASE_URL}modules/es6.symbol.jsx",
-  "{BASE_URL}modules/es6.symbol/index.jsx",
-  "{BASE_URL}modules/es6.symbol.ts",
-  "{BASE_URL}modules/es6.symbol/index.ts",
-  "{BASE_URL}modules/es6.symbol.tsx",
-  "{BASE_URL}modules/es6.symbol/index.tsx",
-  "{BASE_URL}modules/es6.symbol.ls",
-  "{BASE_URL}modules/es6.symbol/index.ls",
-  "{BASE_URL}modules/es6.symbol.json",
-  "{BASE_URL}modules/es6.symbol/index.json",
-  "{BASE_URL}modules/es6.symbol",
+  "{BASE_URL}/blob/path/modules/es6.symbol.js",
```

Comment:
Expected change, since the original test wasn't passing a root path which is now required to make this work